### PR TITLE
#154080094 Fix Login Button

### PR DESCRIPTION
**Pull request by jimmykimani**
#### What does this PR do?
FIx Login Button
#### Description of Task to be completed?
Fix Login Button
#### Where should the reviewer start?
N/A
#### How should this be manually tested?
- Start the project 
- Access `http://localhost:8080/`
- You should see the login page.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
screen shot 2018-01-12 at 5 07 35 pm

### CODE REVIEW

File: src/pages/Login/LoginPage.scss [hunk 1]
```diff
@@ -27,7 +26,7 @@
             .title {
                 margin-bottom: 0.3rem;
                 font-weight: 200;
-                // font-size: 23px;
+                font-size: 20px;
```
Comment (andela-hms) [inline src/pages/Login/LoginPage.scss]: Please change this to rem

File: src/pages/Login/LoginPage.scss [hunk 2]
```diff
@@ -39,16 +38,15 @@
...
-                padding: 0rem;
-                height: 100%;
+                height: 80%;
                 box-sizing: border-box;
             }
 
             .button {
                 margin: 1rem;
-                border: 0.1rem solid map-get($grey, 100);
+                border: 1px solid map-get($grey, 100);
```
Comment (andela-hms) [inline src/pages/Login/LoginPage.scss]: Another one

**Comment by Mbarak-Mbigo** [comment]
Kindly rebase

File: src/pages/Login/LoginPage.scss (hunk 1 above)
Comment (jimmykimani) [reply src/pages/Login/LoginPage.scss]: Noted

**Comment by Mbarak-Mbigo** [comment]
@jimmykimani kindly rebase

**Pull request merged**